lib/find_non_repeated_letter.rb

```diff
@@ -9,6 +9,20 @@
 
 
 def find_first_non_repeated_letter(str)
-  #write your method here
-  return true
+  i = 0
+
+  if str[i] != str[i+1]
+	  return str[0]
+  else
+	  while i < str.length
+		  if str[i] == str[i+1]
+			  i += 1
+		  elsif str[i+1] == str[i+2]
```

Comment:
Can you say more about your inclusion of this `elsif` block? What was your thought process for including it? If you comment it out, does your method still function properly? Why or why not?